media/js/devreg/lookup-tool.js

```diff
@@ -51,6 +51,75 @@ require(['prefetchManifest']);
         $(this).next('button').removeClass('disabled');
     });
 
+    $('.add-group button').click(_pd(function() {
+        var button = $(this);
+        var select = button.parent().parent().find('select');
```

Comment:
Instead of using `.parent().parent()` use `.parents()` with the right selector. That way it'll be more robust.